README.md

```diff
@@ -1,28 +1,32 @@
 # FizzySteamyMirror
 
-This is a community maintained repo forked from [RayStorm](https://github.com/Chykary/FizzySteamyMirror). Mirror [docs](https://mirror-networking.com/docs/Transports/Fizzy.html).
+This is a community maintained repo forked from **[RayStorm](https://github.com/Chykary/FizzySteamyMirror)**. 
 
-FizzySteamyMirror brings together [Steam](https://store.steampowered.com/) and [Mirror](https://github.com/vis2k/Mirror) utilising Async of a Steam P2P network transport layer for **Mirror**.
+Mirror **[docs](https://mirror-networking.com/docs/Transports/Fizzy.html)** and the official community **[Discord](https://discord.gg/N9QVxbM)**.
+
+FizzySteamyMirror brings together **[Steam](https://store.steampowered.com)** and **[Mirror](https://github.com/vis2k/Mirror)** utilising Async of a Steam P2P network transport layer for **Mirror**.
 
 ## Dependencies
-**Note: SteamWorks.NET is packed with this transport.**
+If you want an easy import, skip the steps bellow & download the latest **[unitypackage](https://github.com/Chykary/FizzySteamyMirror/releases)**, **it has SteamWorks.Net already included**.
+
+**Note: If you already have SteamWorks.Net in your project, you might need to delete either your import or the one included in the FizzySteamyMirror [unitypackage](https://github.com/Chykary/FizzySteamyMirror/releases).**
 
 Both of these projects need to be installed and working before you can use this transport.
-1. [Steamworks.NET](https://github.com/rlabrecque/Steamworks.NET) FizzySteamyMirror relies on Steamworks.NET to communicate with the [Steamworks API](https://partner.steamgames.com/doc/sdk). **Requires .Net 4.x**  
-2. [Mirror](https://github.com/vis2k/Mirror) FizzySteamyMirror is also obviously dependant on Mirror which is a streamline, bug fixed, maintained version of UNET for Unity.
+1. **[SteamWorks.NET](https://github.com/rlabrecque/Steamworks.NET)** FizzySteamyMirror relies on Steamworks.NET to communicate with the **[Steamworks API](https://partner.steamgames.com/doc/sdk)**. **Requires .Net 4.x**  
+2. **[Mirror](https://github.com/vis2k/Mirror)** FizzySteamyMirror is also obviously dependant on Mirror which is a streamline, bug fixed, maintained version of UNET for Unity.
 
 ## Setting Up
 
-1. Install Mirror (Requires Mirror 8.0+) from the Unity asset store or grab a unitypackage here: **[Download Mirror](https://github.com/vis2k/Mirror/releases)**

```

Comment:
It's not recommend that you import the **unitypackage** from the **Mirror** GitHub.